test/functional/rpc_packages.py

```
@@ -12,17 +12,22 @@
 from test_framework.messages import (
     tx_from_hex,
 )
+from test_framework.p2p import P2PTxInvStore
 from test_framework.script import (
     CScript,
     OP_TRUE,
 )
 from test_framework.util import (
     assert_equal,
+    assert_fee_amount,
```

Ruff (0.12.2)

`test_framework.util.assert_fee_amount` imported but unused

Remove unused import: `test_framework.util.assert_fee_amount`

(F401)